_includes/banner.html

```diff
@@ -1,4 +1,9 @@
-<div class="banner">
-    <h1>Women Coding Community</h1>
-    <h3>Home Page TBD</h3>
-</div>
+<header class="banner">
+    <div class="overlay-image position-absolute w-100 h-100"></div>   
+    <div class="overlay-dark position-absolute w-100 h-100"></div>
```

Comment:
```suggestion
    <div class="banner-title"></div>
```
Could you create a class for that? just to be simple and clean the html.

Reply:
Ah thanks, I just realised they’re actually redundant Bootstrap classes since I ended up including the `position`, `weight` and `height` for the overlays in the already-custom `overlay-image` and `overlay-dark` classes anyway. 

Going forward, should we generally avoid the built-in Bootstrap classes and prioritise updating the variables, then creating custom classes? If so, are we using a naming convention like BEM for CSS classes?

Reply:
I think we want to move away from bootstrap later on and use more modern React, Tailwind CSS, etc. :) but that's still up for discussion.